src/transaction-types.ts

```diff
@@ -1,5 +1,5 @@
-import type { Bytes } from './bytes';
-import type { Hex } from './hex';
+import type { Bytes } from './bytes.js';
```

Comment:
Is this necessary?

Reply:
Looks like it isn't. Fixed: https://github.com/MetaMask/utils/pull/185/commits/87accddac1b51fc42419be8e9bb6b3cac1aea129

Reply:
I made file extensions optional in `ts-bridge` to make the transition to `NodeNext` easier.

Reply:
@Mrtenz Oh good call on that! Definitely makes it easier to transition.